app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
@iNecas sounds fine to me.

@ares the reason we're considering Foreman::Logging was that we already have such a module, so I'm not sure we should create one with such a similar name.